Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractTouch.cs

```diff
@@ -28,6 +28,8 @@ public class VRTK_InteractTouch : MonoBehaviour
         public event ObjectInteractEventHandler ControllerTouchInteractableObject;
         public event ObjectInteractEventHandler ControllerUntouchInteractableObject;
 
+        public bool triggerOnStaticObjects = false;
```

Comment:
group this with the other public inspector variables (so just stick it above the event initalisation)